_pages/webextensions.md

```diff
@@ -18,13 +18,13 @@ Add-ons provide a safe, simple, and powerful way to personalize your browsing ex
 
 __Impact:__
 
-* 125 developers are introduced to and building WebExtensions either online or at regional events.
+* 125 developers are introduced to and building browser either online or at regional events.
 * 50 add-ons tagged with mozactivate17 in addons.mozilla.org.
-* 50 social media posts (Twitter, Facebook, blogs, etc.) about WebExtensions add-ons referencing #MozActivate
+* 50 social media posts (Twitter, Facebook, blogs, etc.) about that reference extension development and #Mozactivate.
```

Comment:
Sure -- I just want to eliminate 'WebExtensions add-ons.' :) Want it to just say #MozActivate?